### DISCUSSION TITLE
Unable to access email on song.expand?.user?.email

### DISCUSSION THREAD

**Discussion by adampatterson** [pocketbase/pocketbase#2273] [Q&A] [answered]
I'm messing around with Svelte and have a number of songs posted by users. I have a simple Gravitar component that worked while I was logged in. 

`song.expand?.user?.email` = my email 🤘

Once logged out the email is undefined.

`song.expand?.user?.email` = undefined 😭

```
const resultList = await pb.collection('music').getList(1, 50, {
       sort: 'created',
       expand: 'user',
});
```

Is there a way that I can access the email while not logged in? I'd think this has implications for many things like if a user commented on a post and you wanted to notify that user or If I want to show their Gravitar 😄

**Comment by ganigeorgiev** [comment, marked as answer]
Guest users can access other users email only if it is marked public (aka. `emailVisibility` is set to `true`; and if they have view API access to the auth collection).

**Comment by adampatterson** [comment]
@ganigeorgiev Thanks, I saw it under the Users settings but it wasn't obvious how to enable it.

Is this something that can be set when a user is created?

**Reply by ganigeorgiev** [reply]
> Is this something that can be set when a user is created?

Yes, the `emailVisibility` flag can be set as part of the other regular record fields.

**Comment by adampatterson** [comment]
Thanks, My Gravitars work 😄
